openbas-api/src/test/java/io/openbas/rest/PayloadApiTest.java

```diff
@@ -158,4 +279,20 @@ private PayloadCreateInput getExecutablePayloadCreateInput() {
     input.setExecutableArch(Endpoint.PLATFORM_ARCH.x86_64);
     return input;
   }
+
+  private PayloadCreateInput getCommandLinePayloadCreateInput() {
+    PayloadCreateInput input = new PayloadCreateInput();
```

Comment:
**nitpick**: Maybe use a fixture?

Reply:
Ah, I see that this is the *Input DTO-type object, not the actual model. This is probably why I haven't used the fixture in the first place, which returns a Payload model (not a PayloadInput DTO). I'll see if I can "fixturise" this.